hedera-node/hapi-fees/src/main/java/module-info.java

```diff
@@ -22,4 +22,5 @@
     requires com.google.protobuf;
     requires org.apache.commons.lang3;
     requires com.google.common;
+    requires com.hedera.hashgraph.protobuf.java.api;
```

Comment:
This is added because IntelliJ showed errors, but the gradle build still succeeded

Reply:
I wonder if you removed this line and rebuilt the intellij caches if the error would go away?

Reply:
No it still shows errors. Not sure if its my IntelliJ. But the `./gradlew assemble` passes fine.